docs/io/radar/sdk/Radar.RadarGeocodeCallback.html

```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
my apologizes, these doc files all look like new `.kt` files that I added to the project.. 🙈

Reply:
All good, we really need to start generating docs on release instead of in PRs